.github/workflows/release.yml

```
@@ -181,9 +182,10 @@ jobs:
           --acl "public-read" \
           --quiet
 
+
   tag-docker-latest:
-    name: Tag Docker Hub images with release version and latest
-    runs-on: ubuntu-22.04
+    name: Tag ${{ matrix.project_name }} image with release version and latest
+    runs-on: ubuntu-24.04
```

Would probably be good to add a permissions block here like the [build-inified.yml ](https://github.com/bitwarden/self-host/blob/24713e80e9b5c4c8d1148a6a426eafafb5691599/.github/workflows/build-unified.yml#L26 ) one so that it explicitly grants `packages:write` for when we start adding the global section and reduce repo permissions settings in the near future.

```suggestion
    runs-on: ubuntu-24.04
    permissions:
      id-token: write
      packages: write
      security-events: write
```

---

Good point. I added a couple of permissions blocks.